spaceapi/src/lib.rs

```diff
@@ -151,7 +169,7 @@ pub struct Sensors {
 
 #[derive(Debug, Clone)]
 pub struct PeopleNowPresentSensor {
-    pub value: u32,
+    pub value: i64,
```

Comment:
Changed to simplify the type conversion. We now only support 64 bit signed integers, 64 bit floats and booleans. That's all data types I could find in the SpaceAPI documentation (bit length is not specified there).